addOns/replacer/src/main/java/org/zaproxy/zap/extension/replacer/ExtensionReplacer.java

```diff
@@ -136,6 +138,56 @@ private static boolean contains(String original, String match, Pattern p) {
     }
 
     private static String replace(String original, String match, Pattern p, String replacement) {
+        LOGGER.debug("Static Replacement function.");
+        int specialStart = replacement.indexOf("{{");
+        LOGGER.debug("Special start: " + String.valueOf(specialStart));
```

Comment:
Use paremeterised logging, the explicit conversions to string are not needed too.